terraform/deploy/modules.tf

```diff
@@ -65,6 +65,7 @@ module "concourse_web" {
   cognito_domain        = module.cognito.outputs.user_pool_domain
   cognito_issuer        = module.cognito.outputs.issuer
   cognito_name          = module.cognito.outputs.name
+  prometheus_cidr_block = local.cidr_block[local.environment].mon-master-vpc
```

Comment:
can you stick to terraform standards and use underscores instead of hyphens

Reply:
All VPC's are defined with hyphens, not a choice of mine.

Reply:
This is referring to a list of CIDRs that is being pulled from the param store when running `make bootstrap` to configure this file. The entries within that param store entry are using hyphens, and this is referring directly to one of those values - so it's beyond the scope of this PR to fix the param store if that's what we want/need.